core-bundle/src/Resources/config/services.yml

```diff
@@ -510,6 +510,9 @@ services:
     contao.security.token_checker:
         class: Contao\CoreBundle\Security\Authentication\Token\TokenChecker
         arguments:
+            - '@request_stack'
+            - '@security.firewall.map'
+            - '@security.token_storage'
```

Comment:
Just curious: Why did you not add the new arguments at the end?

Reply:
🤷‍♂️ Just because the new ones are kinda more important, the existing ones are not used anymore if the new ones „match“.

Reply:
Absolutely reasonable from my point of view 😄